checks.d/docker_daemon.py

```diff
@@ -806,6 +806,17 @@ def _parse_blkio_metrics(self, stats):
                 metrics['io_write'] += int(line.split()[2])
         return metrics
 
+    def _is_container_cgroup(self, line, selinux_policy):
+        if line[1] not in ('cpu,cpuacct', 'cpuacct,cpu', 'cpuacct'):
```

Comment:
The PR looks good and can be merged as is, but if you change this line into `if line[1] not in ('cpu,cpuacct', 'cpuacct,cpu', 'cpuacct') or line[2] == '/docker-daemon':` it would also fix #2841 :)